halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/BakeryController.java

```diff
@@ -195,6 +195,7 @@
     UpdateRequestBuilder builder = new UpdateRequestBuilder();
     builder.setSeverity(severity);
 
+    // TODO(lwander): Would be good to indicate when an added base image id conflicts with an existing base image id.
```

Comment:
Yeah, I've been thinking about this. It might be best to seed haylard with the existing images rather than have them defined in rosco-web/config/rosco.yml. Once you define any images here they'll override what's in rosco-webo/config/osco.yml anyhow.

WDYT?